microscopium/screens/__init__.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import absolute_import
-from . import myofusion
+from . import myofusion, cellomics
 
-__all__ = ['myofusion']
+__all__ = ['myofusion', 'cellomics']
 
-d = {'myofusion': {'index': myofusion.filename2coord,
-                   'fmap':  myofusion.feature_map}}
+d = {'myofusion': {'index': myofusion.filename2id,
+                   'fmap':  myofusion.feature_map},
```

Comment:
I was wondering about this myself as I fiddled with this file. When I say "screens", do I mean "individual screens" or "screen types"? I think originally I meant the former, but the latter now makes more sense. However the file actually includes feature definitions created specifically for this screen. Number of nuclei per cell, for example, makes no sense in a screen with no cell fusion.

What I propose to do:

- change this to "myores" because "myofusion" causes a lot of headaches.
- open an issue to separate the screen-type and screen-id concepts more clearly across the API.